makefile

```diff
@@ -1,20 +1,58 @@
-# Lists all `make` targets.
-help:
+# Targets marked '# PRIVATE' will be hidden when running `make help`.
+# They're helper targets that you probably only need to know about
+# if you've got as far as reading the makefile.
+
+# Lists common targets.
+# Also the default task (`make` === `make help`).
+help: # PRIVATE
```

Comment:
No, that's clear! For me, I just would find it very strange to run `make help` and not see `help` as an option